src/ai/backend/manager/models/gql.py

```diff
@@ -292,6 +303,11 @@ class Queries(graphene.ObjectType):
         is_active=graphene.Boolean(),
     )
 
+    group_node = graphene.Field(
+        GroupNode, id=graphene.String(required=True), description="Added in 24.03.0."
+    )
```

Comment:
If `AsyncNode` is to handle Relay Global ID, how about setting `id` field to `AsyncNode` itself?

Reply:
I have tried it, but it needs so much work. so I deprioritized this task

Reply:
Let's try this in next PR!